### PR TITLE
next/ecf96e49

### PR DESCRIPTION
**Pull request by stevekinney**
- **feat(components): add Steps wizard indicator and progress reduced-motion regression tests**
- **Address review committee feedback (round 1)**

---

> [!NOTE]
> **Low Risk**
> Low risk: primarily adds a new presentational component and test-only CSS audits, with minimal impact beyond new exports and bundled styles.
> 
> **Overview**
> Adds a new **`Steps`** wizard step indicator component, including its public exports (`src/index.ts`, package `exports`), shared stylesheet aggregation, and dedicated `steps.css` supporting horizontal/vertical layouts and forced-colors handling.
> 
> Introduces `Steps` unit tests covering a11y semantics (`aria-current="step"`, completed-step announcements), prop behavior (orientation/labels), edge cases (empty/clamped indices), and keyed reordering stability.
> 
> Extends `Progress` tests with a reduced-motion regression audit that parses `progress.css` to assert indeterminate bar/ring animations are disabled inside `@media (prefers-reduced-motion: reduce)`.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 761439b1d057fd08b0a66390932da6918602f8ac. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

**Comment by stevekinney** [comment]
@copilot please review.

File: packages/components/src/components/steps.svelte [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        <span class="cinder-steps__marker" aria-hidden="true">
+          {#if isComplete}
+            <Check class="cinder-steps__check" />
+          {:else}
+            <span class="cinder-steps__index">{index + 1}</span>
+          {/if}
+        </span>
+        <span class="cinder-steps__body">
+          {#if isComplete}
+            <span class="cinder-steps__sr-only">{completedLabel}</span>
```
Comment (stevekinney) [inline packages/components/src/components/steps.svelte:85]: This change removes the only separator between the hidden completion label and the visible step label. With the current markup, the accessible text becomes `CompletedSet up profile...` because there is no whitespace text node between the sibling spans. Please preserve the verbatim `completedLabel` value, but add a separator in the markup outside the prop value itself so the announcement stays readable.
